src/components/Modals/CourseMenu.vue

```diff
@@ -110,8 +110,7 @@ export default Vue.extend({
       const itemWidthCompact = 232; // width of a compact semester div in px
 
       const grid = document.getElementsByClassName('semesterView-content')[0];
-      // @ts-ignore
-      const gridStyle = grid.currentStyle || window.getComputedStyle(grid);
+      const gridStyle = window.getComputedStyle(grid);
```

Comment:
currentStyle is not a valid field in a dom element, so it always fallback to `window.getComputedStyle(grid)`